### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by Udokaamos**
Please verify

### CODE REVIEW

**Comment by Udokaamos** [comment]
image

**Comment by Udokaamos** [comment]
image

**Comment by daddychukz** [comment]
Great start!
Take note of the following:
>- Re-arrange your folder structure to have `styles`, `scripts` folders, and rename `img` to `images`.
>- Include relevant files in appropriate directories after creating them (e.g style.css should be in `styles` folder)
>- Ensure proper commit messages. Let your commit message reflect what you actually did in the change you made. Just a summary e.g `Added a new header component`. Do that against `first commit`

File: styles/style.css [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    background:#35424a;
+    color:#ffffff;
+    padding-top:30px;
+    min-height:70px;
+    border-bottom: #caf334 3px solid;
+}
+
+#showcase{
+    min-height:400px;
+    background:url('../img/showcase.jpg') no-repeat 0 -400px;
```
Comment (daddychukz) [inline styles/style.css]: You already changed the folder name so effect the change here...

**Review by daddychukz — CHANGES_REQUESTED** [review]
Left a comment

**Review by daddychukz — APPROVED** [review]

**Pull request merged**